vm-setup/vars/packages_installation.yml

```diff
@@ -0,0 +1,95 @@
+packages_installation:
+  ubuntu:
+    packages:
+     - crudini 
+     - curl 
+     - dnsmasq 
+     - figlet 
+     - golang 
+     - nmap 
+     - patch 
+     - psmisc 
+     - python-pip 
+     - wget
+     - libvirt-bin 
+     - libvirt-clients 
+     - libvirt-dev 
+     - jq 
+     - nodejs 
+     - golang-go 
+     - unzip 
+     - yarn 
+     - genisoimage 
+    libvirt:
+       - qemu-kvm
+       - libguestfs-tools
+       - gir1.2-polkit-1.0
+       - libpolkit-agent-1-0
+       - libpolkit-backend-1-0
+       - libpolkit-gobject-1-0
+    pip:
+     - ansible==2.8.2
+     - lolcat 
+     - yq 
+     - virtualbmc 
+     - python-ironicclient 
+     - python-ironic-inspector-client 
+     - lxml 
+     - netaddr 
+     - requests 
+     - setuptools 
+     - libvirt-python 
+  centos:
```

Comment:
Some of the common packages listed under `centos` don't exist for RHEL8.  The ones I hit were:

```
        "No package crudini available.",
        "No package figlet available.",
        "No package python-pip available.",
        "No package python-requests available.",
        "No package python-setuptools available.",
        "No package python-ironicclient available.",
        "No package python-ironic-inspector-client available.",
        "No package python-openstackclient available."
```

None of those are hard requirements, though.

How about instead of `centos` and then separate lists under `centos.libvirt` for rhel7 vs rhel8, let's just have 2 top level lists, `rhel7` and `rhel8`.  We can drop these from the rhel8 list.

Reply:
@namnx228 would you like to work on this?  or would you like me to do this part?

Reply:
Hi @russellb, sorry for me late reply,
Let me do this part. 
However, Are you sure that if we drop these from rhel8 list, the dev-env still works ?

Reply:
yeah, don't worry about it.  i just need the playbook to run successfully on rhel 8.  if these are needed on rhel 8, they'll have to be installed some other way, and i'll deal with that separately

Reply:
Ok, I will remove them in the next commit